docs/sources/k6/next/using-k6/javascript-typescript-compatibility-mode.md

```diff
@@ -40,40 +40,40 @@ $ k6 run script.js
 
 {{< /code >}}
 
-After v0.53.0 the only difference with base is that `global` (node's global variable) is aliased to the value of `globalThis`.
+After v0.53.0, the only difference from the base mode is that `global`, the Node.js global variable, is aliased to the value of `globalThis`.
```

Comment:
I would merge this sentence with the previous `k6 offers two JavaScript compatibility modes:`. In this way, the reader can join the list of available modes already having the context about the different.

Otherwise, if someone isn't aware of this concept, while it reads the list then they won't get most of it.